lib/mac/bindings.js

```diff
@@ -184,6 +184,69 @@ NobleBindings.prototype.discoverDescriptors = function(identifier, serviceIdenti
   characteristic.discoverDescriptors();
 };
 
+NobleBindings.prototype.read = function(identifier, serviceIdentifier, characteristicIdentifier) {
+  var characteristic = this._characteristicForIdentifier(identifier, serviceIdentifier, characteristicIdentifier);
+
+  characteristic.readValue();
+
+  //can we get isNotification from corebluetooth?
+  characteristic.once('valueUpdate', function(value, error) {
```

Comment:
So the [Yosemite binding](https://github.com/sandeepmistry/noble/blob/master/lib/mac/yosemite.js#L522) is using `false` now, because `args.kCBMsgArgIsNotification` is undefined.  I'm ok with undefined for now we can update the read me accordingly, to say something like true or false if known, otherwise undefined.